shared/chat/conversation/container.tsx

```diff
@@ -27,7 +27,9 @@ export class Conversation extends React.PureComponent<SwitchProps> {
     this.props.selectConversation()
   }
   _onWillBlur = () => {
-    this.props.deselectConversation()
+    if (Container.isPhone) {
+      this.props.deselectConversation()
```

Comment:
i think you want this below, but really just make this be a no op in the dispatch , we never want it to run on tablet right?